src/components/Fieldset/Fieldset.stories.mdx

```diff
@@ -0,0 +1,96 @@
+import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs/blocks';
+import { action } from '@storybook/addon-actions';
+import Fieldset from './Fieldset';
+
+<Meta title="Fieldset" component={Fieldset} />
+
+# Fieldset
+
+A basic uncontrolled input component that for either a `textArea` or `input` element depending on the `isMultiline` prop.
+Rendered inside an associated `label` tag. The `inputRef` prop can be used to find the component's `value`.
```

Comment:
Don't need to talk about markup or the inner workings of the component here. Just what it does and how I can use it